UML_vantuz/FIrstUMLLab/FIrstUMLLab/Subway.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace FIrstUMLLab
+{
+    class Subway : UndergroundTransport
+    {
+        public Turnstiles Turnstiles { get; private set; }
```

Comment:
plural form means that you have to declare List<Turnstile>, do not complicate your code